examples/fhir/MN NBS FHIR Order Message.json

```diff
@@ -0,0 +1,765 @@
+{
+  "resourceType": "Bundle",
+  "id": "b4efef3a-749c-457d-956b-568e22768bf3",
+  "identifier": {
+    "value": "b4efef3a-749c-457d-956b-568e22768bf3"
+  },
+  "type": "message",
+  "timestamp": "2023-05-06T05:29:16.000-05:00",
+  "entry": [
+    {
+      "fullUrl": "MessageHeader/3b5e0ea5-f204-4cb1-8027-7adf07d8065b",
+      "resource": {
+        "resourceType": "MessageHeader",
+        "id": "3b5e0ea5-f204-4cb1-8027-7adf07d8065b",
+        "meta": {
+          "tag": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/v2-0103",
+              "code": "P",
+              "display": "Production"
+            }
+          ]
+        },
+        "eventCoding": {
+          "system": "http://terminology.hl7.org/CodeSystem/v2-0003",
+          "code": "O21",
+          "display": "OML - Laboratory order"
+        },
+        "source": {
+          "name": "Epic",
+          "endpoint": "1.2.840.114350.1.13.145.2.7.2.695071"
+        },
+        "sender": {
+          "reference": "Organization/centracare.com",
+          "type": "Organization"
+        },
+        "destination": [
+          {
+            "name": "NATUS",
+            "endpoint": "http://natus.health.state.mn.us",
+            "receiver": {
+              "reference": "Organization/2.16.840.1.114222.4.1.10080",
+              "type": "Organization"
+            }
+          }
+        ],
+        "focus": [
+          {
+            "reference": "ServiceRequest/34685122-eb17-44c0-aa2c-6a67461dcb73",
+            "type": "ServiceRequest"
+          },
+          {
+            "reference": "ServiceRequest/742d6734-e14c-45b3-af73-f1ee919b3e19",
+            "type": "ServiceRequest"
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "Provenance/3eb5944e-c7da-44d4-80dd-0763c63175c0",
+      "resource": {
+        "resourceType": "Provenance",
+        "id": "3eb5944e-c7da-44d4-80dd-0763c63175c0",
+        "recorded": "2023-05-06T05:29:16.000-05:00",
+        "agent": [
+          {
+            "who": {
+              "reference": "Device/d33c30a0-3ea9-4464-8a51-a197eac9a375",
+              "type": "Device"
+            }
+          }
+        ],
+        "target": [
+          {
+            "reference": "Patient/09a7a7cc-7456-4f09-bdf6-ebc730902fd2",
+            "type": "Patient"
+          }
+        ],
+        "activity": {
```

Comment:
this should probably be `transform` rather than the HL7 message type:  https://hl7.org/fhir/valueset-provenance-activity-type.html

Reply:
@jorg3lopez, @halprin Do we know if `message` is an RS requirement? I copied this from the demographics-only lab_order.json file.

Reply:
In the `Bundle`'s `type` field?  I _think_ it is required.  But it's been a while.  I bet we could re-ask RS why some these requirements are there.

Reply:
John to ask about in ETOR slack channel